sql/dd/impl/bootstrap/bootstrapper.cc

```diff
@@ -1673,6 +1676,25 @@ bool sync_meta_data(THD *thd) {
   return false;
 }
 
+// Helper guard used in update_properties, to be sure
+// that encryption will get set back before
+// update_properties exits.
+struct Target_encryption_guard {
+ public:
+  Target_encryption_guard(const Object_table *object_table)
+      : m_object_table(object_table)
+      : set_encryption(object_table->is_target_encrypted) {}
```

Comment:
is_target_encrypted should be called as function right? is_target_encrypted"()"?

Reply:
I wonder how this worked at all :scream:

Reply:
yeah, scary, fixed.